azerothcore/azerothcore-wotlk
# fix(Core\Scripts):Jiklik's bats from phase 2 now won't engage players…

… directly and will fly



## Changes Proposed:
-  
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/834

## SOURCE:


## Tests Performed:

- Engage Jiklik and change to phase 2 (50% of her total health) and see that the bats spawned during this phase will come down to players and meele them
- 


## How to Test the Changes:


1. Engage Jiklik and change to phase 2 (50% of her total health) and see that now the bats spawned during this phase will keep flying and throw missiles accordingly.
2.
3.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
